### ISSUE TITLE
"Pure-JS" verbiage is misleading

### ISSUE THREAD

**Issue by solumos** [orls/ccv-purejs#11]
The Canvas dependency relies on Cairo (C library) so it's misleading to say that this package is a "pure-js" implementation.

**Comment by miketwalker** [comment]
Agreed, was super excited when I saw this hoping it could be used in AWS Lambda but it's not possible due to the Cairo dependency.

**Comment by solumos** [comment]
@miketwalker I actually got it working on Lambda, but it's not very fun. You have to compile the dependencies on an EC2 instance using the same AMI that Lambda runs.

I was able to get it working with [these notes](https://github.com/Automattic/node-canvas/issues/680#issuecomment-179968875) + some trial and error. Basically, you need to build the `*.so` files on the AMI and package them with your Lambda build.
